packages/@groupby/elements-search-driver-plugin/src/search-driver-plugin.ts

```diff
@@ -137,6 +148,21 @@ export default class SearchDriverPlugin<P = Record> implements Plugin {
       products: mappedRecords,
     };
   }
+
+  /**
+   * Dispatches a search tracker event.
+   *
+   * @param results The results from the search response.
+   * @param originValue The search event's origin.
+   */
+  dispatchSearchTrackerEvent(results: Results, originValue: string): void {
+    const origin: SendableOrigin = { [originValue]: true };
+    const beaconSearchPayload: BeaconSearchPayload = {
+      results,
+      origin,
+    };
+    this.core[this.eventsPluginName].dispatchEvent(BEACON_SEARCH, beaconSearchPayload);
+  }
```

Comment:
It's worth noting that this is completely duplicated between the `SearchDriverPlugin` and the `SaytDriverPlugin`. I believe there is other functionality as well. We should think about how our plugins can share functionality.